new/LIT6608Commentary.xml

```diff
@@ -49,7 +51,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography"/><!---->
+          <div type="bibliography">
+            <listRelation>
+              <relation name="dcterms:creator" active="Deuteronomy" passive="PRS6978Meherka"></relation>
```

Comment:
active="LIT6608Commentary"? or active="LIT6608Commentary#Deuteronomy"? and the rest?